File: samps/contests/tc1/config/ISumitRMSec-3/submissions/accepted/isumit.cs
/*
 *
 * File:    isumit.cs
 * Purpose: to sum the positive from stdin
 * pc2@ecs.csus.edu at http://www.ecs.csus.edu/pc2
 *
 * $Id: hello.cs 3159 2015-09-22 20:39:11Z boudreat $
*/
using System;
 
public class iSumit
{
    static public void Main ()
    {
        int sum = 0;
        string str;
        while((str = Console.ReadLine()) != null) 
        {
            int num = int.Parse(str);
            if (num > 0)
            {
                sum = sum + num;
            }
        }
        Console.WriteLine ("The sum of the integers is {0}",sum);
    }
}

File: samps/contests/pointscoring/config/a/submissions/accepted/ISumit.java

// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
import java.io.*;

//
// File:    ISumit.java
// Purpose: to print the sum of the positive integers from stdin

public class ISumit {
    public static void main(String[] args) {
        try {
            BufferedReader br = new BufferedReader(new InputStreamReader(System.in), 1);

            String line;
            int sum = 0;
            int rv = 0;
            while ((line = br.readLine()) != null) {
                rv = new Integer(line.trim()).intValue();
                if (rv > 0)
                    sum = sum + rv;
            }
            System.out.print("The sum of the integers is ");
            System.out.println(sum);
        } catch (Exception e) {
            System.out.println("Possible trouble reading stdin");
            System.out.println("Message: " + e.getMessage());
        }
    }
}